client/my-sites/checkout/utils.ts

```diff
@@ -135,3 +144,49 @@ export function getAffiliateCouponLabel(): string {
 	// translators: The label of the coupon line item in checkout
 	return translate( 'Exclusive Offer Applied' );
 }
+
+export function useGetWpcomPlanTotalIfPaidMonthly( products: Array< ResponseCartProduct > ): {
+	[ key: string ]: number | undefined;
```

Comment:
Is it possible to narrow the return type? Something like:

```
[ key in ( typeof PlanSlugs )[ number ] ]?: number | null;`
```
or

```
Record< ( typeof PlanSlugs )[ number ], number | null >
```

🤔

Reply:
I've changed it to be more specific.